client/locations-list-map.js

```diff
@@ -550,168 +809,155 @@ function onFilterChange(data, prefix, idx, selected, filters) {
+      text: itemFilter.name,
+      selected: itemFilter.isSet,
+    });
   }
 
-  $(".map-container").show();
-
-  map = new google.maps.Map(element);
-  secondaryCluster = new MarkerClusterer(map, [], {
-    clusterClass: 'secondarycluster',
-    imagePath: '/images/markercluster/m',
-    minimumClusterSize: 5,
-    zIndex: 1,
+  const ppeNeededSelect = new Selectr('#ppe-needed-select', {
+    customClass: 'ftm-select',
+    data: acceptedItems,
+    multiple: true,
+    searchable: false,
+    placeholder: $.i18n('ftm-ppe-needed'),
   });
-  primaryCluster = new MarkerClusterer(map, [],
-    {
-      imagePath: '/images/markercluster/m',
-      minimumClusterSize: 5,
-      zIndex: 2
-    });
 
-  primaryCluster.addListener('click', (e) => {
-    sendEvent('map', 'click', 'primaryCluster');
+  ppeNeededSelect.on('selectr.select', (option) => {
+    onFilterChange(data, 'acceptItems', option.idx, true, filters);
+    sendEvent('filters', 'acceptItems', option.value);
   });
 
-  secondaryCluster.addListener('click', (e) => {
-    sendEvent('map', 'click', 'secondaryCluster');
+  ppeNeededSelect.on('selectr.deselect', (option) => {
+    onFilterChange(data, 'acceptItems', option.idx, false, filters);
   });
 
-  google.maps.event.addListener(map, 'bounds_changed', () => {
-    const mapBounds = map.getBounds();
-
-    if (mapBounds && currentViewportCenter) {
-      const currentLat = mapBounds.getCenter().lat();
-      const currentLng = mapBounds.getCenter().lng();
+  const facilityTypes = [];
 
-      if (currentLat !== currentViewportCenter.lat || currentLng !== currentViewportCenter.lng) {
-        refreshList(data, filters);
-      }
+  for (const item of Object.keys(filters.orgTypes)) {
+    const orgType = filters.orgTypes[item];
+    facilityTypes.push({
+      value: orgType.value,
+      text: orgType.name,
+      selected: orgType.isSet,
+    });
+  }
 
-      currentViewportCenter = {
-        lat: currentLat,
-        lng: currentLng
-      }
-    }
-  });
+  if (facilityTypes.length > 0) {
+    const facilityTypeSelect = new Selectr('#facility-type-select', {
+      customClass: 'ftm-select',
+      data: facilityTypes,
+      multiple: true,
+      searchable: false,
+      placeholder: $.i18n('ftm-facility-type'),
+    });
 
-  const mapBounds = map.getBounds();
+    facilityTypeSelect.on('selectr.select', (option) => {
+      onFilterChange(data, 'orgTypes', option.idx, true, filters);
+      sendEvent('filters', 'orgTypes', option.value);
+    });
 
-  if (mapBounds) {
-    const mapCenter = mapBounds.getCenter();
-    currentViewportCenter = {
-      lat: mapCenter.lat(),
-      lng: mapCenter.lng()
-    };
+    facilityTypeSelect.on('selectr.deselect', (option) => {
+      onFilterChange(data, 'orgTypes', option.idx, false, filters);
+    });
+  } else {
+    $('#facility-type-select').parent().remove();
   }
+}
 
-  showMarkers(data, filters);
+function getDataJsonUrl(filename) {
+  return `/${filename}`;
+}
```

Comment:
Do we really need a function to prepend a slash to a path?

Reply:
no... I think this used to do more. Removed.